app/actions/service_route_binding_delete.rb

```diff
@@ -4,27 +4,29 @@
 module VCAP::CloudController
   module V3
     class ServiceRouteBindingDelete < V3::ServiceBindingDelete
-      def initialize(service_event_repository)
+      def initialize(user_audit_info)
         super()
-        @service_event_repository = service_event_repository
+        @user_audit_info = user_audit_info
       end
 
       private
 
-      attr_reader :service_event_repository
+      def binding_event_repository
+        Repositories::ServiceGenericBindingEventRepository.new('service_route_binding')
+      end
 
       def perform_delete_actions(binding)
-        record_audit_event(binding)
+        binding_event_repository.record_delete(
+          binding,
+          @user_audit_info
+        )
+
         binding.destroy
         binding.notify_diego
       end
 
-      def record_audit_event(binding)
-        service_event_repository.record_service_instance_event(
-          :unbind_route,
-          binding.service_instance,
-          { route_guid: binding.route.guid },
-        )
+      def perform_start_delete_actions(binding)
```

Comment:
I found this rename slightly obscure... `record_audit_event` is a little more explicit than `perform_start_delete_actions`. The latter makes me believe there are some things we need to do before deleting when in fact we are only logging the audit event. 

I can see how this is better in terms of symmetry with the `perform_delete_actions` though.

Reply:
The idea here is that every binding can choose to do whatever they need as part of recording an async operation. It just happens to be just auditing for now. 
I thought this naming would keep it open to extentions